CoEpi/repo/SymptomRepo.swift

```diff
@@ -0,0 +1,30 @@
+import Foundation
+
+protocol SymptomRepo {
+    func symptoms() -> [Symptom]
+
+    func submitSymptoms(symptoms: [Symptom])
+}
+
+class SymptomRepoImpl: SymptomRepo {
+
+    func symptoms() -> [Symptom] {
+        [
+            Symptom(name: "Fever"),
```

Comment:
I have the same hardcoded strings, in a follow up we should try to centralize them https://github.com/Co-Epi/app-ios/pull/12/files#diff-0edb6816c821d71dbbd3b88f4d79479cR13

Reply:
The repo is currently the central. As a further step the backend could deliver them.